src/PayID/xrp-pay-id-client.ts

```diff
@@ -11,4 +11,18 @@ export default class XRPPayIDClient extends PayIDClient {
   constructor(public readonly xrplNetwork: XRPLNetwork) {
     super(`xrpl-${xrplNetwork}`)
   }
+
+  /**
+   * Retrieve the XRP address associated with a PayID.
+   *
+   * Note: Addresses are always in the X-Address format.
+   * @see https://xrpaddress.info/
+   *
+   * @param payID The payID to resolve for an address.
+   * @returns An address representing the given PayID.
+   */
+  async xrpAddressForPayID(payID: string): Promise<string> {
+    const result = await super.addressForPayID(payID)
+    return result.address
```

Comment:
Does this not also just return the classic address?  like even though `addressForPayID` returns the full address details?  Oh wait, this is what the future PR is going to do... it's going to make sure that this address that you're digging out of the details IS an X-address and therefore incorporates the tag... Just lemme know if I'm wrong about this otherwise approving.

Reply:
No that's exactly the case. I was just doing it in two PRs to make the deltas clear for reviewers.